modules/pom.xml

```diff
@@ -35,6 +35,7 @@
         <module>schedulers</module>
         <module>scripting</module>
         <module>scripting-jruby</module>
+        <module>scripting-plugin</module>
```

Comment:
it doesn't matter that this is not an extension or that it's a plugin. What I meant is that this is the kind of thing that we potentially want to have on a different release lifecycle so that we can evolve it at a different pace